app/controllers/containers_controller.rb

```diff
@@ -56,6 +77,16 @@ def action_permission
       :view
     when 'commit'
       :commit
+    when 'power'
+      :power_compute_resources_vms
```

Comment:
Just `:power` should suffice if you create the `:power_containers` permission, that way you can get rid of `current_permission` too